plugins/woocommerce/package.json

```diff
@@ -33,7 +33,7 @@
 		"env:test": "pnpm env:dev",
 		"env:perf": "pnpm env:dev && pnpm env:performance-init",
 		"preinstall": "npx only-allow pnpm",
-		"postinstall": "rimraf -g vendor/woocommerce && XDEBUG_MODE=off composer install --quiet",
+		"postinstall": "rimraf -g vendor/woocommerce && rimraf -g packages/email-editor && XDEBUG_MODE=off composer install --quiet",
```

Comment:
Can be simplified to `rimraf -g vendor/woocommerce packages/email-editor && XDEBUG_MODE=off composer install --quiet`

Reply:
Nice 👍🏽